tutorials/meshing_of_fractures.ipynb

```diff
@@ -56,10 +56,11 @@
    ]
```

Comment:
<p>Consider renaming this tutorial to e.g. mixed_dimensional_grids.ipynb or fracture_networks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/878/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>